src/react/components/pages/projectSettings/projectForm.test.tsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import ProjectForm from "./projectForm";
+import { mount } from "enzyme";
+import { MockFactory } from "../../../../models/mockFactory";
+import TagColors from "../../common/tagsInput/tagColors.json";
+
+import { Provider } from "react-redux";
+import { BrowserRouter as Router } from "react-router-dom";
+import { IApplicationState } from "../../../../models/applicationState";
+import initialState from "../../../../redux/store/initialState";
+import createReduxStore from "../../../../redux/store/store";
+import ProjectSettingsPage from "./projectSettingsPage";
+
+describe("Project Form Component", () => {
+    let onSubmit: (value: any) => void;
+
+    const mockFactory = new MockFactory();
```

Comment:
If you convert your methods to static you wouldn't need to create an instance... I don't believe you are tracking any state within the object, correct?

Reply:
Correct. I'll move all the mocks to your class and make them static